### PR TITLE
autopkgtest corrections

### PR DESCRIPTION
**Pull request by mathiascode**
- Run autopkgtest on current code instead of master
- Pybuild sets the http proxy to 127.0.0.1:9, which conflicts with our tests that need internet access. Disable tests as part of the build process, since they're defined to run later in debian/tests/control anyway.

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -35,6 +35,9 @@ SOURCE_DIR  = $(abspath $(PACKAGE_DIR)/../)
 #  embedded directly in the Python source...
 VERSION_UPSTREAM_DISTUTILS = $(shell sed -n 's/^version\s*=\s*"\(\S*\)"$$/\1/p' $(SOURCE_DIR)/pynicotine/utils.py)
 
+# Disable tests in pybuild, we run them later...
+export PYBUILD_DISABLE=test
```
Comment (kiplingw) [inline debian/rules:39]: Maybe it would be safer to disable only those tests that depend on network connectivity?

Comment (mathiascode) [reply debian/rules:39]: We can probably do that.

**Comment by mathiascode** [comment]
Not sure if someone's abusing one of GitHub's macOS test servers or something else is going on, but it seems to hit GitHub's rate limit at times when trying to fetch the latest version.

**Comment by mathiascode** [comment]
https://github.com/actions/virtual-environments/issues/602

I'll go ahead and disable the version test on the macOS workflow, if that's ok? @kiplingw

**Comment by kiplingw** [comment]
Sure, but please keep an eye on what's happening with the OS X environment in case someone discovers a workaround.

**Comment by mathiascode** [comment]
Will do.

**Pull request merged**